src/utils.ts

```diff
@@ -2,11 +2,37 @@ import {
   multilineCommentsRE,
   singlelineCommentsRE,
 } from 'vite-plugin-utils'
+import { AcornNode } from './types'
 
 export function isCommonjs(code: string) {
   // Avoid matching the content of the comment
   code = code
     .replace(multilineCommentsRE, '')
     .replace(singlelineCommentsRE, '')
   return /\b(?:require|module|exports)\b/.test(code)
-}
\ No newline at end of file
+}
+
+
+export function simpleWalk(
+  ast: AcornNode,
+  visitors: {
+    [type: string]: (node: AcornNode, ancestors: AcornNode[]) => void | Promise<void>,
```

Comment:
A type with a return value of `Promise<void>` con't be processed here.

```diff
- [type: string]: (node: AcornNode, ancestors: AcornNode[]) => void | Promise<void>,
+ [type: string]: (node: AcornNode, ancestors: AcornNode[]) => void,
```

Reply:
😉